quickstarts/Embeddings.ipynb

```diff
@@ -92,37 +107,46 @@
     },
     {
       "cell_type": "code",
-      "execution_count": null,
+      "execution_count": 5,
       "metadata": {
-        "id": "J76TNa3QDwCc"
+        "id": "J76TNa3QDwCc",
+        "colab": {
+          "base_uri": "https://localhost:8080/",
+          "height": 34
+        },
+        "outputId": "17678b10-0cfd-4551-8a19-dd3164aadfd9"
       },
       "outputs": [
         {
-          "name": "stdout",
           "output_type": "stream",
+          "name": "stdout",
           "text": [
-            "[0.04703258, -0.040190056, -0.029026963, -0.026809 ... TRIMMED]\n"
+            "[0.013168523, -0.008711934, -0.046782676, 0.000699 ... TRIMMED]\n"
           ]
         }
       ],
       "source": [
         "text = \"Hello world\"\n",
-        "result = genai.embed_content(model=\"models/embedding-001\", content=text)\n",
+        "result = genai.embed_content(model=\"models/text-embedding-004\", content=text)\n",
```

Comment:
If you use `output_dimensionality=10` then you don't need the complicated print out.